components/kyma-environment-broker/internal/storage/dbmodel/operation.go

```diff
@@ -9,9 +9,10 @@ import (
 
 // OperationFilter holds the filters when listing multiple operations
 type OperationFilter struct {
-	Page     int
-	PageSize int
-	States   []string
+	InstanceFilter *InstanceFilter
```

Comment:
potentially there is room for supporting all filters, not only `InstanceIDs`, but at the moment that would be non trivial amount of work.